src/cc65/datatype.c

```diff
@@ -66,6 +66,7 @@ Type type_int[]         = { TYPE(T_INT),    TYPE(T_END) };
 Type type_uint[]        = { TYPE(T_UINT),   TYPE(T_END) };
 Type type_long[]        = { TYPE(T_LONG),   TYPE(T_END) };
 Type type_ulong[]       = { TYPE(T_ULONG),  TYPE(T_END) };
+Type type_bool[]        = { TYPE(T_INT),    TYPE(T_END) };
```

Comment:
What should happen to `type_bool` if `_Bool` is introduced? Then it becomes `TYPE(T_INT)` or we just need to figure it out later?

Reply:
It would use a new `T_BOOL` (that is  `T_TYPE_BOOL | T_CLASS_INT | T_SIGN_UNSIGNED | T_SIZE_BOOL`, and probably a `BOOL_BITS` that equals `1`. It would need special handling like `char` and `enum`.